rfcs/text/0000-deprecate-log-original.md

```diff
@@ -0,0 +1,106 @@
+# 0000: Name of RFC
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+This RFC supersedes issue #841 which implies breaking changes therefore the RFC Process is indicated
+
+The request is to consolidate `log.original` and `event.original` by removing `log.original`, since these are almost equivalent in nature. (One) justification for preserving `event.original` is that not all events are logs.
+
+Breaking changes may be:
```

Comment:
We should clarify the removal of `log.original` _will_ be considered a breaking change since the field's being removed from the schema.

But these points are still great to capture about possible migration/mitigations for users impacted.

Reply:
applied